tasks/cockpit-tasks

```diff
@@ -113,14 +113,30 @@ function run_one_task() {
     fi
 }
 
+function run_from_queue() {
+    cd "$XDG_CACHE_HOME"/cockpit-project/bots
+    # Consuming directly from queue, commands have bots/ prepended
+    [ -e bots ] || ln -s . bots
+    if /usr/bin/timeout 12h /bin/sh -c "set -ex; ./run-queue --amqp \"$AMQP_SERVER\""; then
+        work_done=1
+    fi
+}
+
 # on mass deployment, avoid GitHub stampede
 sleep $(shuf -i 0-120 -n 1)
 
 # Perform a couple of tasks or waits, then restart
 for i in $(seq 1 30); do
     work_done=
 
-    update_repo  "$XDG_CACHE_HOME"/cockpit-project/bots             https://github.com/cockpit-project/cockpit
+    update_repo  "$XDG_CACHE_HOME"/cockpit-project/bots             https://github.com/cockpit-project/bots
+
+    # 80% to try to consume from queue so other tasks don't get starved
```

Comment:
<bikeshed> 20% for tasks is probably waaaaaay to much. We need to run them about once a day, and this loop iterates ~ 100 times every day if it could. In fact, it would be sufficient to call the `run_one_task` exactly once at startup, as the pod restarts after 30 iterations the loop anyway. This makes the script easier to understand, more predictable, and avoids the shuffling.

But if you prefer, this can happen in a separate PR (even before landing this one).